src/main/java/org/isf/accounting/service/AccountingIoOperations.java

```diff
@@ -287,8 +287,21 @@ public boolean deleteBill(
 	 * @param dateTo the high date range endpoint, inclusive.
 	 * @return a list of retrieved {@link Bill}s.
 	 * @throws OHServiceException if an error occurs retrieving the bill list.
+	 * @deprecated use {@link #getBillsByDate(GregorianCalendar, GregorianCalendar)}
 	 */
+	@Deprecated
 	public ArrayList<Bill> getBills(GregorianCalendar dateFrom, GregorianCalendar dateTo) throws OHServiceException {
+		return getBillsByDate(dateFrom, dateTo);
+	}
+
+	/**
+	 * Retrieves all the {@link Bill}s for the specified date range.
+	 * @param dateFrom the low date range endpoint, inclusive.
+	 * @param dateTo the high date range endpoint, inclusive.
+	 * @return a list of retrieved {@link Bill}s.
+	 * @throws OHServiceException if an error occurs retrieving the bill list.
+	 */
+	public ArrayList<Bill> getBillsByDate(GregorianCalendar dateFrom, GregorianCalendar dateTo) throws OHServiceException {
```

Comment:
Nice the idea to add @deprecate, for this method I would suggest `getBillsBetweenDates` ('byDate' would suggest one single date)